sklearn/tests/test_naive_bayes.py

```diff
@@ -335,7 +345,6 @@ def test_discretenb_predict_proba():
         assert_almost_equal(np.sum(clf.predict_proba([X[1]])), 1)
         assert_almost_equal(np.sum(clf.predict_proba([X[-1]])), 1)
         assert_almost_equal(np.sum(np.exp(clf.class_log_prior_)), 1)
-        assert_almost_equal(np.sum(np.exp(clf.intercept_)), 1)
```

Comment:
Revert and add comment to remove in 0.26 when `intercept_` is deprecated.

Reply:
Done!